frank-doc-doclet/src/main/java/org/frankframework/frankdoc/model/FrankElement.java

```diff
@@ -195,46 +200,73 @@ void handleAncestorMethod(FrankClass ancestorClass) {
 		}
 	}
 
-	private void handlePossibleParameters(FrankClass clazz) {
+	private List<ParsedJavaDocTag> parseParameterJavadocTags(FrankClass clazz) {
+		return parseJavadocTags(clazz, JAVADOC_PARAMETER);
+	}
+
+	private String parseParametersJavadocTag(FrankClass clazz) {
 		try {
-			this.meaningOfParameters = Utils.substituteJavadocTags(clazz.getJavaDocTag(JAVADOC_PARAMETERS), clazz);
+			return Utils.substituteJavadocTags(clazz.getJavaDocTag(JAVADOC_PARAMETERS), clazz);
 		} catch(FrankDocException e) {
 			log.error("Error parsing the meaning of parameters", e);
 		}
-		assembleParsedJavaDocTags(clazz, JAVADOC_PARAMETER, p -> this.specificParameters.add(p));
+
+		return null;
 	}
 
-	private void handlePossibleForwards(FrankClass clazz) {
-		assembleParsedJavaDocTags(clazz, JAVADOC_FORWARD, p -> this.forwards.add(p));
+	private List<ParsedJavaDocTag> parseForwardJavadocTags(FrankClass clazz) {
+		return parseJavadocTags(clazz, JAVADOC_FORWARD);
+	}
+
+	private List<QuickLink> parseSeeJavadocTags(FrankClass clazz) {
+		return clazz.getAllJavaDocTagsOf(JAVADOC_SEE).stream()
+			.map(value -> {
+				final var matcher = JAVADOC_SEE_PATTERN.matcher(value);
+
+				if (!matcher.matches()) {
+					return null;
+				}
+
+				String label = matcher.group(2);
+				String url = matcher.group(1);
+
+				return new QuickLink(label, url);
```

Comment:
Kwestie van stijl denk ik maar ik vind het leesbaarder om hier een aparte methode van te maken, bv `convertToQuicklink` oid.

Reply:
Ik ben persoonlijk niet zo'n voorstander van allemaal kleine functies die maar op 1 plek gebruikt worden. Ik vind het vooral niet erg als het binnen een map gedaan wordt. Als het team dit liever niet wilt zien pas ik het uiteraard aan

Reply:
Ik weet niet wat het team wil, maar ik weet wel wat ik vind :-) Een methode die maar op 1 plek wordt aangeroepen zie ik niet als een probleem, het bevordert de leesbaarheid en je documenteert door een duidelijke method name ook gelijk je code (omdat je aan de naam ziet wat de methode doet). Ik heb nog wel een leuk boek hierover als je het interessant vindt ;-)

Reply:
Ik denk dat het er leesbaarder door wordt. Je kan een lambda meegeven; `this::toQuickLink` en dan ook makkelijk deze mapping testen met een junittest. String er in, quicklink er uit.